setup.cfg

```diff
@@ -9,12 +9,16 @@ exclude =
   **/*/version.py,
   tests/.tests
 
-[tool:pytest]
+[tool:pytest] --ignore-glob=*version.py
 addopts = --flake8 --black --mypy
```

Comment:
Shouldn't the ignore bit in the previous line be on this line instead?

Reply:
yes. Thanks for catching that.